extensions/shared/external-media/media-browser/index.js

```diff
@@ -55,14 +79,11 @@ function MediaBrowser( props ) {
 			}
 
 			setSelected( newSelected );
+			setFocused( index );
```

Comment:
I think trying to keep track of what is currently focused in this way is going to end up being really difficult to get right in the end 😬 I don't think we need to keep track of it overall, just respond to the keydown listener from the media item.

Reply:
Updated in https://github.com/Automattic/jetpack/pull/16118/commits/9aff1e247def4b907bb29a63058ccda176bbb17d